pineappl_py/src/import_only_subgrid.rs

```diff
@@ -2,10 +2,75 @@ use super::subgrid::PySubgridEnum;
 
 use numpy::{PyReadonlyArray1, PyReadonlyArray3};
 use pineappl::import_only_subgrid::ImportOnlySubgridV1;
+use pineappl::import_only_subgrid::ImportOnlySubgridV2;
 use pineappl::sparse_array3::SparseArray3;
-
+use pineappl::subgrid::Mu2;
 use pyo3::prelude::*;
 
+/// PyO3 wrapper to :rustdoc:`pineappl::import_only_subgrid::ImportOnlySubgridV2 <import_only_subgrid/struct.ImportOnlySubgridV2.html>`
+///
+/// **Usage**: `pineko`
+#[pyclass]
+#[derive(Clone)]
+#[repr(transparent)]
+pub struct PyImportOnlySubgridV2 {
+    pub(crate) import_only_subgrid: ImportOnlySubgridV2,
+}
+
+impl PyImportOnlySubgridV2 {
```

Comment:
Remove this block.

Reply:
Fine: since the attribute is `pub(crate)` anyhow, you do not need another `pub(crate)` constructor.

Reply:
[605fd90](https://github.com/NNPDF/pineappl/pull/222/commits/605fd90d088b4c6a384dac6b0c162460328393e7)